### PR TITLE
Release/5.8.0

### PR DESCRIPTION
**Pull request by stefaniacardenas**
### Added
- Public: Changes to allow hybrid desktop/mobile devices  with environment facing cameras (e.g. Surface Pro) to use the `useLiveDocumentCapture` feature (BETA feature)
- Internal: Analytics can now be disabled via the `disableAnalytics` option
- Internal: Test coverage for snapshot feature
- Internal: Send additional properties to back-end in `sdkMetadata` object
  - `isCrossDeviceFlow` (true|false)
  - `deviceType` (mobile|desktop)
  - `captureMethod` (live|html5)

### Changed
- Internal: Use `v2/snapshots` endpoint to upload additional selfie frames.
- Internal: Split Confirm component into multiple files.
- UI: Accessibility - Update font colours and weight following DAC Audit report feedback
- Internal: Pushing `dist` files to S3 and publishing the release to NPM has been automated using GitHub Actions
- Internal: Improve UI tests stability when looking for and clicking on UI elements
- Public: Documentation should use `v3` for API endpoints and include links to migration guide.

### Fixed
- Public: Fixed bug where iPads on iOS13 were detected as desktop devices.
- Public: Made fallback error message appropriate for both face and document verification
- Public: Fixed video recording in liveness capture step not working for Firefox >= 71
- Internal: Fix flaky modal UI tests
- Public: Fixed bug where blob was not handled correctly when an upload event was fired on IE11
- Public: Fixed camera permission screen layout issue on desktop Safari where buttons disappears below view height
- Public: Prevent "submit" event from being emitted when selecting a document

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +46,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Public: Added a troubleshooting section to the documentation with details about solving CSP related issues
 - UI: Added selfie intro screen
 - UI: Option to send cross device secure link using QR code (**Note:** *changes introduced with this UI update include possible breaking changes for integrators with custom translations or copy*)
+- Public: Added a `userAnalyticsEvent` to existing analytics calls for integrators to listen for.
```
Comment (stefaniacardenas) [inline CHANGELOG.md]: @kopijunkie @bartzet I can see that this line might have been added under the wrong release header by mistake. I will move it under `5.8.0`

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,37 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [next-version]
+
+## [5.8.0]
```
Comment (bartzet) [inline CHANGELOG.md]: When do we add the date?

File: src/components/utils/unit_tests/onfidoApi.test.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      sendMultiframeSelfie(snapshotData, selfieData, jwtToken, url, onSuccess, onError)
+      mockXHR.onload(); // First XHR has been mocked
+      await Promise.resolve();
+      mockXHR.onload(); // Second XHR has been mocked
+      expect.assertions(2); // There are two assertion in this test case because onSuccess is calling `expect` too
+      expect(mockXHR.send.mock.calls.length).toBe(2) // Two request send() methods have been called
+    })
+  })
+
+  // TODO: Re-enable this tests once the backend endpoint is in production
```
Comment (bartzet) [inline src/components/utils/unit_tests/onfidoApi.test.js]: We are going to have the backend in production ready after this platform release, aren't we?

Comment (stefaniacardenas) [reply src/components/utils/unit_tests/onfidoApi.test.js]: They will let us know by Thursday

**Comment by rfreitas** [comment]
Travis automatic deployment: https://946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-946-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://946-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**